django_redis/cache.py

```diff
@@ -150,3 +150,14 @@ def lock(self, *args, **kwargs):
     @omit_exception
     def close(self, **kwargs):
         self.client.close(**kwargs)
+
+    @omit_exception
+    def touch(self, key, timeout=None, version=None):
+        try:
+            return self.client.touch(key, timeout=timeout, version=version)
+        except ConnectionInterrupted as e:
```

Comment:
Isn't this exception already handled the `omit_exception`? Why does it need to be repeated?